segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/SystemJournal.java

```diff
@@ -139,23 +183,47 @@
     @Getter
     private final ChunkedSegmentStorageConfig config;
 
+    private final GarbageCollector garbageCollector;
+
+    private final Supplier<Long> currentTimeSupplier;
+
     private final AtomicBoolean reentryGuard = new AtomicBoolean();
 
     /**
      * Constructs an instance of {@link SystemJournal}.
-     *
-     * @param containerId   Container id of the owner container.
+     *  @param containerId   Container id of the owner container.
      * @param chunkStorage  ChunkStorage instance to use for writing all logs.
      * @param metadataStore ChunkMetadataStore for owner container.
+     * @param garbageCollector GarbageCollection instance.
+     * @param currentTimeSupplier Function that supplies current time.
```

Comment:
Please document the time unit it returns. Is it millis or nanos (or something else)?

Reply:
done.